docs/metrics/lead-cycle-time.md

```diff
@@ -0,0 +1,31 @@
+# Lead and cycle time metrics
+
+Pulse calculates lead and cycle time based on issue state changes in Jira.
+
+![Lead time versus cycle time](images/lead-cycle-time.png)
+
+-   [Lead time](#lead-time)
+-   [Cycle time](#cycle-time)
+
+<!--TODO Some things to take into consideration into the definition of the metrics:
+
+* what happens when an issue is reopened?
+* what happens when an issue changes type?
+* what happens when an issue changes project?
```

Comment:
- The metrics are only computed for closed issues, so if an issue is reopened it won't be considered for the lead or cycle time metrics.
- It will be reflected on our side. Note that this will not affect lead or cycle times.
- It will be reflected on our side. Note that this will not affect lead or cycle times, since the issue still has the same ID (it just changed the project). So all the issue history transitions to the new project.

Reply:
I added these details on how we calculate the metrics in specific scenarios. Please have a look:

https://deploy-preview-82--docs-pulse.netlify.app/metrics/lead-cycle-time/

We can add more detail later if we remember some other scenario or edge case. :+1:

Reply:
Yeah, I think it's good. If necessary we'll iterate.